library/src/compute/calc.rs

```diff
@@ -962,6 +964,40 @@ pub fn quo(
     })
 }
 
+#[comemo::memoize]
+#[func]
+/// Returns a random number
+///
+///
+/// ## Example
+/// ```example
+/// #assert(calc.random(seed: 1) == calc.random(seed: 1))
+/// #assert(calc.random(seed: 1) != calc.random(seed: 2))
+/// ```
+///
+/// Display: Random
+/// Category: Calculate
+/// Returns: integer
+pub fn random(
```

Comment:
Could you create a random float function too?

Reply:
Added